sdk/src/gateway/layerzero.ts

```diff
@@ -85,31 +85,6 @@ export interface SendParams {
     oftCmd: Hex;
 }
 
-export namespace SendParams {
-    export function toBytes(params: SendParams): Hex {
```

Comment:
got rid of this because its cant be used, since abi params need to all be encoded together